scripts/distribution/opentestnet-client.Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN --mount=type=ssh ./build-binaries.sh "collector" release && \
     sha256sum genesis.dat && \
     cp genesis.dat /build-project/
 
-FROM 192549843005.dkr.ecr.eu-west-1.amazonaws.com/concordium/node-dashboard:0.1.0-alpha as node-dashboard
+FROM 192549843005.dkr.ecr.eu-west-1.amazonaws.com/concordium/node-dashboard:0.1.1 as node-dashboard
```

Comment:
Is this tagging done in the jenkins job as a parameter?

Reply:
The node-dashboard job takes a version parameter for the tag, yes. I suppose it could be helpful to parametrise this job also at some point.